README.md

```diff
@@ -130,21 +130,21 @@ Here is what it should look like when you make changes:
 Using SSH tunnels into your CSCI account, you can access the Gitlab repos and webpage from anywhere. I'm doing this so that I can have my own setup away from the CSCI computers, since they cause me constant issues, such as with my quota. Here's how I do it.
 
 #### Gitlab Webpage
-1. `ssh -N -L 127.0.0.1:8080:gitlab.csci.viu.ca:80 exstu@pup10.csci.viu.ca -J exstu@csci.viu.ca`
+1. `ssh -N -L removed`
 2. Access `127.0.0.1:8080` in your web browser.
 
 #### Gitlab Repository Access
 1. Put your Gitlab SSH key in your `~/.ssh` directory in your personal machine.
 2. Add your ssh key to your ssh agent e.g. `eval $(ssh-agent) && ssh-add ~/.ssh/gitlab-ed25519`
-3. Setup tunnel with proxyjump to pup server: `ssh -N -L 127.0.0.1:22:gitlab.csci.viu.ca:22 exstu@pup10.csci.viu.ca -J exstu@csci.viu.ca`
+3. Setup tunnel with proxyjump to pup server: `ssh -N -L removed`
 
 Now you can do normal git commands, but you have to use `127.0.0.1` to access them. For example, cloning a repo:
 -  `git clone git@127.0.0.1:<username>/tekku.git`
 - Add remote to upstream: `git remote add git@127.0.0.1:oleander/tekku.git`
 
 ## View Dolphin Webpages from Off Campus

```

Comment:
I think this section, and "Connecting PHP Backend to Database" can be completely removed.